### ISSUE TITLE
Keybase: Files need to be rekeyed

### ISSUE THREAD

**Issue by MasterRoot24** [keybase/client#4685]
So, when I installed macOS Sierra, I erased my HDD and performed a clean installation. Unfortunately, before I did this, I didn't think to do anything with Keybase so when I tried to set it back up again after the OS install, I wasn't able to as there was already a device in my account with the name "Joe's MacBook Pro".

I opted to completely reset my account, as I thought it would be a quick and easy way back to having my account setup again on my Mac with the name that I wanted to use. I was wrong, but that's another issue (#1952).

Now, I've reset my account, used a different name for my Mac and I've performed a full reinstall of Keybase. However:

```
joenyland@Joes-MBP ~ $ ls /keybase/private/masterroot24\ \(files\ before\ account\ reset\ 2016-09-30\)/
ls: cannot open directory '/keybase/private/masterroot24 (files before account reset 2016-09-30)/': Permission denied
```

...followed by the notification:
> Keybase: Files need to be rekeyed

I don't think there's anything of value in my KBFS, so can I just delete "`files before account reset...`" somehow?

**Comment by strib** [comment]
Hi @MasterRoot24.  Yeah we don't have any way to recover your personal private folder after an account reset.  You can remove the folder from your "favorites" list (the ones that show up in `ls /keybase/private` or in the Keybase GUI) with a command like `rmdir "/keybase/private/masterroot24 (files before account reset 2016-09-30)/"`.

You can also do it via the Keybase app, by clicking the folder and then clicking "Ignore folder".

**Comment by MasterRoot24** [comment]
Hi @strib Ok, no worries. I didn't really expect there would be as I couldn't see how I would be able to decrypt them if I had reset my keys, so that's what I expected.

Regarding the "`files before account reset...`" folders - I just tried to `rmdir` them now and they're gone, but I could have sworn that before opening this issue I had tried to `rm -rf` them and got an error about permissions (obviously tried again with `sudo` but still no dice). Strange how it worked when I tried `rmdir` as you suggested; as far as I know rmdir and `rm -r` don't operate any differently...


Anyway, they're gone now so issue solved. Thanks!

**Comment by strib** [comment]
`rm -rf` actually tries to delete all the individual files inside the directory before deleting the directory itself (which you don't have permission to do since you reset your account), which `rmdir` does not do. Usually file systems don't allow `rmdir` on non-empty directories, but KBFS has a special exception in this case. Glad it worked!
